pynest/nest/lib/hl_api_info.py

```diff
@@ -84,18 +84,15 @@ def helpdesk():
         msg = "Sorry, the help index cannot be opened. "
         msg += "Did you run 'make html' before running 'make install'?"
         raise FileNotFoundError(msg)
-
+    
+    helpdesk_url = "file://" + help_fname
+    
     # Under Windows systems webbrowser.open is incomplete
     # See <https://bugs.python.org/issue8232>
     if sys.platform[:3] == "win":
-        os.startfile(helpfile)
+        os.startfile(helpdesk_url)
```

Comment:
@jougs I just tested  under Win10 and `webrowser.open_new()` works even there (Py 3.7.9), so I suggest we remove the special case for Windows here. It is anyways academic since we currently don't support NEST on Windows, and this entire mechanism requires a running NEST.

Reply:
OK. Thanks for testing. I've removed the workaround in 9798864.